languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -83801,7 +83813,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                   <example>Sirius B ist der lichtschwache Begleiter von Sirius A.</example>
                 </antipattern>
                 <antipattern>
-                  <token postag="UNKNOWN">São</token>
+                  <token postag="UNKNOWN" regexp="yes">São</token>
                   <token postag="UNKNOWN"/>
                   <example>São Francisco</example>
```

Comment:
`São Francisco` could be in global_spelling.txt. There are many places named `São Francisco`, and it cannot be confused with other langauges.

Reply:
Yes, thank you! I will add it.